example/install_src.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+
+PWD=$(pwd)
+MYDIR=${1:-$PWD}
+if [ ! -d $MYDIR ]; then
+    echo "[Error] working directory=$MYDIR doesn't exist"
+    exit -1
+fi
+
+pushd $MYDIR
+
+WORKDIR=$(pwd)  # Absolute Path
+
+
+set -e
+set -x
+
+BRANCH=${2:-master} # can be a tag like 1.0.0
+
+
+
+RACKHD_DEP_REPOS=("on-core" "on-tasks" "di.js" )
+RACKHD_MAIN_REPOS=("on-http" "on-taskgraph" "on-dhcp-proxy" "on-tftp" "on-syslog" )
+RACKHD_NODE_REPOS=("${RACKHD_DEP_REPOS[@]}" "${RACKHD_MAIN_REPOS[@]}")
+RACKHD_OTHER_REPOS=( "on-wss" "on-tools" "on-imagebuilder" "RackHD")
+REPOS=( "${RACKHD_NODE_REPOS[@]}"  "${RACKHD_OTHER_REPOS[@]}" )
+
+GITHUB="https://github.com/RackHD"
+
+
+echo "[Info] Clone RackHD repos, and checkout to  branch ${BRANCH}"
+
+
+for r in ${REPOS[@]}; do
+    rm ${r} -rf
+    git clone ${GITHUB}/${r}.git
+    pushd ${r}
+    git fetch --all --prune --tags
+    if [ -z $BRANCH ]; then
+        git checkout ${BRANCH}
+    fi
+    popd
+done
+
+for r in ${RACKHD_NODE_REPOS[@]}; do
+    pushd ${r}
+    npm install
+    popd
+done
+
+echo "[Info] Make common static directory & generate Docs"
+HTTP_STATIC_FOLDER=on-http/static/http
+TFTP_STATIC_FOLDER=on-tftp/static/tftp
+mkdir -p $HTTP_STATIC_FOLDER
+mkdir -p $TFTP_STATIC_FOLDER
+mkdir -p on-http/static/http/common
+pushd on-http
+npm install apidoc
+npm run apidoc
+npm run taskdoc
+popd
+
+
+
+echo "[Info] Download Static Images"
+HTTP_BASE_URL=https://bintray.com/artifact/download/rackhd/binary/builds/
+TFTP_BASE_URL=https://bintray.com/artifact/download/rackhd/binary/builds/
```

Comment:
👎 
seems API changed:

it should be 
https://dl.bintray.com/rackhd/binary/builds